backstop/tests/text-field.html

```diff
@@ -146,7 +159,7 @@ <h2>Default</h2>
 
     <h2>Inline</h2>
     <section id="demo-inline">
-      <label class="iui-input-container iui-inline">
+      <label class="iui-input-container iui-with-message iui-inline-label">
```

Comment:
I assume `iui-with-message` is not needed here, cause we do not display any message...